pyproject.toml

```diff
@@ -47,7 +47,7 @@ dependencies = [
     "pytest-subtests>=0.2.1,<0.8.0",
     "pytest>=4.6.4,<8",
     "PyYAML>=5.1,<7.0",
-    "requests>=2.22,<=2.28.1",
+    "requests>=2.22,<=2.28.2",
```

Comment:
I think the current approach is too restrictive, maybe we can cap on the minor version, eg, ‘<2.29’

Reply:
Good idea, updated